### PR TITLE
Add support for oci8

### LINKED ISSUE #165: Add support for Oracle DB (oci)

### PR DESCRIPTION
**Pull request by mlocati**
Closes #165

### CODE REVIEW

**Comment by mvorisek** [comment]
@mlocati ❤️ you

**Comment by mlocati** [comment]
@mvorisek sorry, I'm already married 🤣

File: install-php-extensions [hunk 1]
```diff
@@ -973,6 +980,63 @@ compareVersions() {
...
+			installOracleInstantClient_sdk=https://raw.githubusercontent.com/pwnlabs/oracle-instantclient/master/instantclient-sdk-linux-$installOracleInstantClient_version.0.4.0.zip
+		else
+			installOracleInstantClient_version='19.9'
+			installOracleInstantClient_ic=https://download.oracle.com/otn_software/linux/instantclient/199000/instantclient-basic-linux-$installOracleInstantClient_version.0.0.0dbru.zip
+			installOracleInstantClient_sdk=https://download.oracle.com/otn_software/linux/instantclient/199000/instantclient-sdk-linux-$installOracleInstantClient_version.0.0.0dbru.zip
+		fi
+	else
+		installOracleInstantClient_client=client64
+		if test "$DISTRO" = 'alpine'; then
+			# I couldn't find a way to install libresolv.so on Linux - let's use an old version that doesn't require it
```
Comment (mvorisek) [inline install-php-extensions:1000]: @mlocati https://github.com/atk4/image/blob/89433b7b29010b9f7046498ba745904af439e476/Dockerfile.template#L54 is this a solution for it?

Comment (mlocati) [reply install-php-extensions:1000]: IDK, I'll give it a try

Comment (mlocati) [reply install-php-extensions:1000]: It seems to work: see #220 (and thanks!)

**Pull request merged**